<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="TaubmanBasic"
           tests="1"
           failures="1"
           errors="0"
           skipped="0"
           time="50.360">
  <properties />
            <testcase name="signUp" time="50.360" classname="ParkCharge.Regression.TaubmanPrepod">
                <failure type="java.lang.IllegalStateException"
                  message="Cannot get a text value from a numeric cell"
              >
<![CDATA[java.lang.IllegalStateException: Cannot get a text value from a numeric cell
  org.apache.poi.xssf.usermodel.XSSFCell.typeMismatch(XSSFCell.java:855)
  org.apache.poi.xssf.usermodel.XSSFCell.getRichStringCellValue(XSSFCell.java:299)
  org.apache.poi.xssf.usermodel.XSSFCell.getStringCellValue(XSSFCell.java:251)
  ParkCharge.Regression.TestBase.getData(TestBase.java:64)
  ParkCharge.Regression.TaubmanPrepod.signUp(TaubmanPrepod.java:56)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
  org.testng.TestRunner.privateRun(TestRunner.java:767)
  org.testng.TestRunner.run(TestRunner.java:617)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
  org.testng.SuiteRunner.run(SuiteRunner.java:254)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
  org.testng.TestNG.run(TestNG.java:1057)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
  org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
      </failure>

          </testcase>
  
  
  
</testsuite>
